core/src/runtime/hooks.rs

```diff
@@ -58,13 +58,14 @@ pub struct HookEnv<'a, 'b: 'a> {
 }
 
 pub fn hook_ecrecover(_env: HookEnv, buf: &[u8]) -> Vec<Vec<u8>> {
+    assert_eq!(
+        buf.len(),
+        65 + 33,
```

Comment:
should be 65 + 32? this should be caught if we have ecrecover test?

Reply:
fixed and added test